.gitignore

```diff
@@ -27,3 +27,6 @@ podman-remote*.zip
 podman*.tar.gz
 .idea*
 .vscode*
+contrib/spec/podman.spec
```

Comment:
Do we really want to ignore the spec file?  Will this cause issues when building the rpm if you don't remember to go remove it first?

Reply:
Is this generated during rpm_build. On any code repository `git status` should return nothing after you run any kind of tools. Build artifacts are not supposed to be tracked, regardless of their nature. The real source of this file is `podman.spec.in`.

Reply:
Maybe someone can explain me the failed gating job, is not clear to me what failed there.

Reply:
@ssbbarnea thanks for the info, I'd forgotten that libpod has a '*.in' rather than a '*.spec' file.  That make sense.

IDK, what's up with the gating for sure, but I do know there's issues with gating running across a number of the projects.